docker-compose.yaml

```diff
@@ -0,0 +1,101 @@
+version: "3.7"
+
```

Comment:
I assume `docker compose up` would bring up the full stack yes?

Reply:
Yes, exactly